utils/perl5lib/Module/ModuleLoader.pm

```diff
@@ -655,15 +655,26 @@ sub loadSoftModules()
 
     my %newbuildenv;
 
+	# ndk: add a loop to see if our above module adjusting _removed_
```

Comment:
We are just about done with the perl module loader.  We rely much much less on environment variables now so I hope we no longer need this kind of hack.